.rubocop.yml

```diff
@@ -0,0 +1,153 @@
+inherit_from: .rubocop_todo.yml
+
+AllCops:
+  Exclude:
+    # Exclude .gemspec files because they are generally auto-generated
+    - '*.gemspec'
+
+# Generally, the keyword style uses a lot of space. This is particularly true when
+# you use case/if statements, in combination with a long-name variable.
+#
+#   invoice_error_message = case error
+#                           when 1 == 1
+#                             do_something
+#                           else
+#                             do_else
+#                           end
+#
+Lint/EndAlignment:
+  AlignWith: variable
+
+# [codesmell]
+Metrics/AbcSize:
+  Enabled: false
+  Exclude:
+    - 'spec/**/*_spec.rb'
+    - 'test/**/*_test.rb'
+
+# [codesmell]
+Metrics/ClassLength:
+  Enabled: false
+  Exclude:
+    - 'spec/**/*_spec.rb'
+    - 'test/**/*_test.rb'
+
+# [codesmell]
+Metrics/MethodLength:
+  Enabled: false
+  Exclude:
+    - 'spec/**/*_spec.rb'
+    - 'test/**/*_test.rb'
+  Max: 15
```

Comment:
I agree that blindly following metrics is not a good idea so having this as a "codesmell" is ok. We can run it from time to time to check the health of the codebase (which for what we usually do should be quite high). That said I think 10 is a better number than 15 for that purpose.